tests/drivers/i2s/i2s_speed/src/test_i2s_speed.c

```diff
@@ -10,6 +10,12 @@
 #include <drivers/i2s.h>
 
 #define I2S_DEV_NAME "I2S_0"
```

Comment:
Seems like these should be renamed to I2S_DEV_NAME_TX, I2S_DEV_NAME_RX

Reply:
Done, thank you